include/seqan3/search/algorithm/detail/search_scheme_precomputed.hpp

```diff
@@ -47,7 +47,7 @@ namespace seqan3::detail
  */
 
 //!\brief Object storing information for a search (of a search scheme).
-//        Number of blocks have to be known at compile time.
+//!       Number of blocks have to be known at compile time.
```

Comment:
Don't structs always need an `\ingroup` statement?
EDIT: I see that it is grouped in Seqach/Algorithm... Do you know how?

Reply:
```
/*!\addtogroup submodule_search_algorithm
```
In line 45

Reply:
Ah ok thanks